### DISCUSSION TITLE
Dark theme on Web

### DISCUSSION THREAD

**Discussion by AnonymousWebHacker** [hyperspacedev/hyperspace#243] [Ideas]
a web version, it should have a dark version theme like the clients

**Comment by alicerunsonfedora** [comment]
The web version is almost identical to the desktop version and contains dark mode. Albeit, this mode is not the _default_ for accessibility reasons, but it does exist. By default, it will try to match the theme of your system, but you can change this in Settings like you would the desktop app by turning off "Match system appearance" and then toggling dark mode manually.

<img width="1442" alt="image" src="https://user-images.githubusercontent.com/13445064/112066421-af489400-8b3c-11eb-9d27-162d11a864c9.png">
